src/drivers/drv_rgb_led.c

```diff
@@ -240,99 +98,114 @@ void rgb_send(int data) {
     rgb_dma_phase = 3;
 }
 
-// if dshot dma is used the routine is in that file
-#if !defined(USE_DSHOT_DMA_DRIVER) && defined(RGB_LED_DMA) && (RGB_LED_NUMBER > 0)
-
-void DMA1_Channel4_5_IRQHandler() {
-  DMA_Cmd(DMA1_Channel5, DISABLE);
-  DMA_Cmd(DMA1_Channel2, DISABLE);
-  DMA_Cmd(DMA1_Channel4, DISABLE);
-
-  TIM_DMACmd(TIM1, TIM_DMA_Update | TIM_DMA_CC4 | TIM_DMA_CC1, DISABLE);
-  DMA_ClearITPendingBit(DMA1_IT_TC4);
-  TIM_Cmd(TIM1, DISABLE);
-
-  rgb_dma_phase = 0;
-}
-#endif
-
-#else
-
-// sets all leds to a brightness
-void rgb_init() {
-  // spi port inits
-
-  if ((RGB_PIN == LL_GPIO_PIN_13 || RGB_PIN == LL_GPIO_PIN_14) && RGB_PORT == GPIOA) {
-    // programming port used
-
-    // wait until 2 seconds from powerup passed
-    while (time_micros() < 2e6)
-      ;
-  }
-
+void rgb_initIO(void)
```

Comment:
applies to all the internal functions: please stick with snake_case. eg this should be `rgb_init_io`
the old-style `(void)` parameter is not required anymore. please remove.